Helm/opensearch-dashboards/values.yaml

```diff
@@ -59,23 +58,21 @@ securityContext:
   runAsNonRoot: true
   runAsUser: 1000
 
-
 config:
-  dashboards.yml:
+  opensearch_dashboards.yml:
     server:
-        name: dashboards
-        host: 0.0.0.0
-
-    ## Dashboards TLS Config
-        ssl:
-          enabled: false
-          key: /usr/share/dashboards/certs/dashboards-key.pem
-          certificate: /usr/share/dashboards/certs/dashboards-crt.pem
+      name: dashboards
+      host: 0.0.0.0
+
+      ## Dashboards TLS Config
+      ssl:
+        enabled: false
+        key: /usr/share/dashboards/certs/dashboards-key.pem
```

Comment:
Just to mention consistency in my opinion we should change the paths for the certificates and keys as well.

Reply:
Absolutely, I will update it shortly, I unfortunately was not using that in my deployment so I missed it.